tests/pyccel/scripts/test2.json

```diff
@@ -0,0 +1,66 @@
+{
+    "exec": "gfortran",
+    "mpi_exec": "mpif90",
+    "language": "fortran",
+    "module_output_flag": "-J",
+    "debug_flags": [
+        "-fcheck=bounds"
+    ],
+    "release_flags": [
+        "-O3",
+        "-funroll-loops"
+    ],
+    "general_flags": [
+        "-fPIC"
+    ],
+    "standard_flags": [
+        "-std=f2003"
+    ],
+    "mpi": {},
+    "openmp": {
+        "flags": [
+            "-fopenmp"
+        ],
+        "libs": [
+            "gomp"
+        ]
+    },
+    "openacc": {
+        "flags": [
+            "-ta=multicore",
+            "-Minfo=accel"
+        ]
+    },
+    "family": "GNU",
+    "libs": [
+        "-lm"
+    ],
+    "python": {
+        "flags": [
+            "-Wno-unused-result",
+            "-Wsign-compare",
+            "-DNDEBUG",
+            "-g",
+            "-fwrapv",
+            "-O2",
+            "-Wall",
+            "-g",
+            "-fstack-protector-strong",
+            "-Wformat",
+            "-Werror=format-security",
+            "-g",
+            "-fwrapv",
+            "-O2"
+        ],
+        "includes": [
+            "/usr/include/python3.10",
+            "/home/eniddealla/.local/lib/python3.10/site-packages/numpy/core/include"
+        ],
+        "shared_suffix": ".cpython-310-x86_64-linux-gnu.so",
+        "libs": [],
+        "libdirs": [
+            "/usr/lib/python3.10/config-3.10-x86_64-linux-gnu",
+            "/usr/lib/x86_64-linux-gnu"
+        ]
+    }
+}
```

Comment:
Beware, these files are auto-generated by the tests but they should not be committed to the repo. Maybe you have some kind of setting which automatically commits any new file in the folder?